docs/l3/api_flow.md

```diff
@@ -0,0 +1,413 @@
+# Prerequisites (dscp-api-identity-service)
+
+## Setting node aliases 
+
+The values set for each persona are your choice, they should provide a recognisable value in response bodies.
+
+Here we have went with:
+
+- “Member A” to represent an order
+
+- “Member B” to represent a capacity
```

Comment:
MemberA and MemberB represent 'a member/organisation/identity/something making an order/capacity'

Reply:
```suggestion
- “Member A” to represent a persona placing an order

- “Member B” to represent a persona offering capacity
```